proactor/src/main/java/com/iluwatar/proactor/App.java

```diff
@@ -0,0 +1,23 @@
+package com.iluwatar.proactor;
+
+
+/**
+ * Proactor is a software design pattern for event handling in which
+ * long running activities are running in an asynchronous part.
+ *
+ * <p>This pattern simplifies asynchronous application development
+ * by integrating the demultiplexing of completion events and the
+ * dispatching of their corresponding event handlers.
+ */
+public class App {
+  /**
+   * Execute the pattern.
+   *
+   * @param args system input
+   */
+  public static void main(String[] args) throws Exception {
+    var operationProcessor = new AsynchronousOperationProcessor();
+    var client = new Client();
+    client.run(operationProcessor);
```

Comment:
Add comments what's happening here

Reply:
add

Reply:
I don't see any comments yet

Reply:
`When there is an internet service program which needs to deal with many requests at the same time, an unblocking pattern just like proactor pattern is needed.` This example has been added, maybe it didn't show at old commit. [the commit](https://github.com/iluwatar/java-design-patterns/pull/1766/commits/e8649f8f307396da4661b0773d22d3cf4856e586)